src/loader/index.html

```diff
@@ -1 +1 @@
-<span>{$infinite.loaderLabel}</span>
```

Comment:
yes use $infinite

Reply:
Actually, `props` is correct here given how it's used in `infinite-list` (and `infinite-list` shouldn't be depending on it anyway). It appears that `infinite-loader` is actually somewhat useless because all it is is a component that displays its label, so we could remove the loader entirely. You can remove this component if you want, but otherwise, we can take on that work.